modules/manager/controller_setup.go

```diff
@@ -508,13 +511,16 @@ func SetupControllers(mgr manager.Manager, c *Config) (map[string]ControllerDef,
 			return nil, err
 		}
 
+		// REVIEW: Should we define the recorder here, or define it out of the section to allow setting custom metrics in other controllers
+
```

Comment:
I believe it's fine to have it defined outside of this if block (as it is done right now).
```suggestion
```